packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconUnverified from 'terra-icon/lib/icon/IconDiamond';
+import Observation from './common/_Observation';
+import observationPropShape from './proptypes/observationPropTypes';
+import styles from './ClinicalResult.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Result Object with the clinical result data.
+   */
+  resultData: PropTypes.shape({
+    /**
+     *  Systolic Result for blood pressure.
+     */
+    systolic: observationPropShape,
+    /**
+     *  Diastolic Result for blood pressure.
+     */
+    diastolic: observationPropShape,
+  }),
+  /**
+   * Visually hides the unit of measure when presented in a series of side-by-side columns of the same unit.
+   */
+  hideUnit: PropTypes.bool,
+  /**
+   * Whether or not the text should be truncated in display. Restricts clinical result details each to one line.
+   */
+  isTruncated: PropTypes.bool,
+};
+
+const defaultProps = {
+  resultData: {},
+  hideUnit: false,
+  isTruncated: false,
+};
+
+const ClinicalResultBloodPressure = (props) => {
+  const {
+    resultData,
+    hideUnit,
+    isTruncated,
+    ...customProps
+  } = props;
+
+  const isEmpty = (str) => (!str || str.length === 0);
+
+  const decoratedResultDisplay = [];
+
+  const systolicUnitCompare = !isEmpty(resultData.systolic.result.unit) ? resultData.systolic.result.unit.trim().toLowerCase() : null;
```

Comment:
These deep dives into objects are dangerous. Could we mark these values as required in the defined propTypes to better communicate our expectations? The other option I see is to create an Observation class with getters that could dry up some of this validation, but that might be awkward in its own way.

Reply:
Same as above, for all normal results it would be expected to provide an id, but since the same component has built in displays for “error” and known “no data” scenarios, those are some scenarios for which it wouldn’t be needed, going against setting it as required. We had a discussion about it specifically and initial thoughts were to keep it non-required for the time being, monitor usage, and then major version bump later with any other adjustments if it is determined as needed.